pkg/server/container_create.go

```diff
@@ -793,8 +793,7 @@ func generateApparmorSpecOpts(apparmorProf string, privileged, apparmorEnabled b
 	case runtimeDefault:
 		// TODO (mikebrow): delete created apparmor default profile
 		return apparmor.WithDefaultProfile(appArmorDefaultProfileName), nil
-	// TODO(random-liu): Should support "unconfined" after kubernetes#52395 lands.
```

Comment:
unconfined means not applying any profile.
Please reference the implementation in seccomp, and do similar thing.

Reply:
agree

Reply:
agree...